Automattic/wp-calypso
# Add marketing info banner to gifting checkout page

#### Proposed Changes

* This PR adds a marketing info banner to the Gifts checkout page.
* The intention is to increase checkout conversions by providing more information and tying the checkout page back to the original Gifting banner, so the user feels like they're in the right place.
* We try to load the banner quickly to reduce user bouncing.

Before | After
--|--
![gift-order-before](https://user-images.githubusercontent.com/140841/212404214-3438e1fb-d277-4152-9624-a756b043eb71.png)  |  ![gift-order-after](https://user-images.githubusercontent.com/140841/212404194-c23b5b69-a7db-4716-b7df-ab08394e4f2c.png)

#### Testing Instructions



* Load this PR
* Make sure it the Gifting checkout order banner only displays on Gifting checkouts. It should not appear on any other checkout pages.
* Make sure the Gifting checkout banner is responsive and looks good on mobile.
* Makes sure the Gifting banner CSS is specific to our banner and won't affect other areas of the site.

#### Pre-merge Checklist





Related to https://github.com/Automattic/dotcom-forge/issues/1422
